src/components/modal/modal.ts

```diff
@@ -17,6 +17,11 @@ const FOLLOWING = Node.DOCUMENT_POSITION_FOLLOWING | Node.DOCUMENT_POSITION_CONT
 
 @customElement(`${prefix}-modal` as any)
 class BXModal extends HostListenerMixin(LitElement) {
+  /**
+   * The element that had focus before this modal gets open.
+   */
+  private _launcher: Element | null = null;
```

Comment:
I _think_ you can just type this as `Element` and assign to null ...

Reply:
Interesting point - Tried it, but got:

![image](https://user-images.githubusercontent.com/1259051/62574635-eae50a80-b8d3-11e9-8294-77ec93607487.png)

Looking for better way to write `null`-able...

Reply:
Oh weird ... If we have `strictNullChecks` on that may be why ... `Element | null` would be the correct signature for null strictness 👍

Reply:
Uh I see - We have `strict` in `tsconfig.json` and that explains. Thank you for your thoughts here @cal-smith!